CMakeLists.txt

```diff
@@ -60,8 +60,8 @@ include(VMCInstallLibDir)
 #--- Add the packages sources --------------------------------------------------
 #
 if(Geant4VMC_BUILD_G4Root)
+  set(G4Root_BUILD_TEST ${Geant4VMC_BUILD_G4Root_TEST} CACHE BOOL "Build G4Root test")
```

Comment:
Do we need to define a  "cached" variable here? As we use the same pattern to propagate the option set via Geant4VMC_variable also to examples, I'd like to keep the same approach.

Reply:
I believe that without using `CACHE`, the variable is not propagated (at least it was not propagated in my test in a clean environment).

Reply:
Thank you. Indeed, there is a difference between g4root and examples CMake variables: this one is defined in g4root as option while those passed to examples are not redefined. 
And I confirm, without `CACHE` it fails also on my side.